### ISSUE TITLE
You cannot use sortable list and virtual list at same time

### ISSUE THREAD

**Issue by mmis1000** [framework7io/framework7#2849]
* Framework7 version: 3.5.2
* Vue.js version: 2
* Platform and Target: All
* Live Link or JSFiddle: PREFERABLY _(IF YOU WANT YOUR ISSUE TO BE RESOLVED ASAP)_. You can start from forking the following JSFiddle template: [JSFiddle](https://jsfiddle.net/nolimits4web/jcwzqovz/)

### Describe the bug

`Framework7ListItem` does not properly present `f7VirtualListIndex` in rendered element,  
cause `sortable:sort` handler not triggered at all

### To Reproduce
1. Setup `virtual list` and `sortable list` at same time like tutorial
2. Add `sortable:sort` handler to it
3. drag the items to sort them

### Expected behavior
The handler is triggered with the new/old index of dragable items

### Actual Behavior
Nothing happened

### Additional context
Although list presented `f7VirtualList` correctly, the item list does not care about `f7VirtualListIndex` at all, maybe it is a missing feature during version update

**Comment by mmis1000** [comment]
@nolimits4web is the index here works correctly?
As far as I know, the index here is item index in the full list, instead index in the filtered partial list

**Comment by nolimits4web** [comment]
@mmis1000 true, pushed fix. It also can be something like `items.indexOf(item)`

**Comment by nolimits4web** [comment]
This will be available in next update

**Comment by mmis1000** [comment]
And in filtered list, the index number may not be a continuious index list(If I did not read the code wrongly), it can be a parsed array.
If
A LIst [0, 1, 2, 3, 4, 5, 6, 7]
That filtered the odd number
[1, 3, 5, 7]
Then the index would be
2, 4, 6, 8
instead of
2, 3, 4, 5

**Comment by nolimits4web** [comment]
Then just it should be `items.indexOf(item)` and it will be correct
